src/pages/_app.page.tsx

```diff
@@ -0,0 +1,21 @@
+import { QueryClientProvider } from '@tanstack/react-query';
+import { ReactQueryDevtools } from '@tanstack/react-query-devtools';
+import type { AppProps } from 'next/app';
+import { RecoilRoot } from 'recoil';
+
+import queryClient from '../configs/queryClient';
+import '../styles/common.css';
+import '../styles/reset.css';
```

Comment:
```suggestion
import '../styles/reset.css';
import '../styles/common.css';
```

reset보다는 common이 우선적으로 적용되어야하니 순서를 바꿔야할 것 같습니다!